.github/workflows/arc-themes-base-to-new-tag.yml

```diff
@@ -49,12 +49,12 @@ jobs:
 
     steps:
       - name: Checkout repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Setup Node and NPM
         uses: actions/setup-node@v3
         with:
-          node-version-file: .nvmrc
+          node-version: "16"
```

Comment:
I'll accept this, but when we move to fusion 4, I think it's fair to revert these to use the nvm version so we don't have different versions all over the place...

Reply:
Yes, that was my idea too. Having just this set to 16 shouldn't break anything else in the meantime.